akka-stream/src/main/scala/akka/stream/Graph.scala

```diff
@@ -26,5 +26,10 @@ trait Graph[+S <: Shape, +M] {
 
   def named(name: String): Graph[S, M] = withAttributes(Attributes.name(name))
```

Comment:
please change this to addAttributes as well, and remove the `.nest()` from the the implementation of `Flow.withAttributes`